Makefile

```diff
@@ -1,6 +1,6 @@
 PROTO_FILE=modify.proto
 PROTO_GENERATED_FILES_PATH=pkg/rpc
-VERSION="v0.1.3"
+VERSION="v0.2.0"
```

Comment:
Why the minor version bump instead of another patch bump? (ie what is the breaking change that I'm missing?)

Reply:
There are no breaking changes, we simply follow [sem versioning](https://semver.org/). Changes that meaningfully alter behavior (in this case we are explicitly requiring resizer LE) ought to have a minor version bump (`v0.2.0` would have arguably been more appropriate back when follow resizer LE was implemented).

I'm open to bumping up to `v0.1.4` here instead, at your recommendation.

Reply:
My recommendation would be `v0.2.0`, I think it's time